addon/styles/addon.scss

```diff
@@ -95,6 +80,164 @@ $z-index-modal: 2 !default;
     }
+    .nav-column {
+      display: flex;
+      flex: 1;
+      flex-direction: column;
+      padding: 30px;
+
+      &+ .nav-column {
+        border-left: 1px solid rgba($frost-color-white, 0.3);
+      }
+    }
+
+    .nav-section {
+      &+ .nav-section {
+        margin-top: 50px;
+      }
+      .nav-section-header {
+        align-items: center;
+        border-bottom: 1px solid $frost-color-white;
+        color: $frost-color-white;
+        display: flex;
+        flex-direction: row;
+        font-size: $frost-font-l; // Override colors using inline style
+        font-weight: 200;
+        margin-bottom: 30px;
+        padding-bottom: 5px;
+        .pointer {
+          div {
+            display: inline-block;
+          }
+          cursor: pointer;
+        }
+        & .title {
+          display: inline-block;
+          padding-left: 10px;
+          .sub {
+            display: inline-block;
+            font-size: $frost-font-s;
+          }
+        }
+        & .frost-button > .icon {
+          align-items: initial;
+        }
+      }
+
+      .nav-route {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        font-weight: 200;
+        .content {
+          text-align: initial;
+        }
+        .frost-link {
+          border-radius: 3px;
+          cursor: pointer;
+          padding: 3px;
+          transition: box-shadow 0.3s ease;
+          width: 100%;
+          .content {
+            justify-content: initial;
+          }
+          .nav-route-icon {
+            height: 50px;
+            margin-right: 15px;
+            .frost-icon {
+              height: 50px;
+              width: 50px;
+            }
+          }
+          .nav-route-name {
+            border-bottom-color: transparent;
+            border-bottom-style: solid;
+            border-bottom-width: 1px;
+            color: $frost-color-lgrey-3;
+            display: inline-block;
+            font-size: $frost-font-m;
+            transition: color 0.5s ease;
+          }
+          .nav-route-description {
+            color: $frost-color-grey-6;
+            font-size: $frost-font-s;
+            padding-top: 4px;
+            transition: color 0.2s ease;
+          }
+          &:focus {
+            background: rgba(0,0,0,0.2);
+            box-shadow: 0px 0px 6px #fff;
+            outline: none;
+            .nav-route-name {
+              color: $frost-color-white;
+            }
+            .nav-route-description {
+              color: $frost-color-lgrey-2;
+            }
+          }
+          &:hover {
+            .nav-route-name {
+              border-bottom-color: $frost-color-white;
+              color: $frost-color-white;
+            }
+            .nav-route-description {
+              color: $frost-color-lgrey-2;
+            }
+          }
+        }
+
+        &+ .nav-route {
+          margin-top: 25px;
+        }
+      }
+      .frost-icon-add {
```

Comment:
Minor, but I believe this can be targeted a bit more cleanly https://github.com/ciena-frost/ember-frost-core/blob/master/addon/styles/_frost-icon.scss